packages/build-ui/src/main.rs

```diff
@@ -38,6 +39,14 @@ fn App() -> Element {
 fn load_tailwindcss() -> Element {
     rsx! {
         script { src: "https://unpkg.com/@tailwindcss/browser@4" }
+        style { r#type: "text/tailwindcss",
+            r#"
+                @theme {{
+                    --color-primary: #30D4A0;
+                    --color-background: #171717;
+                }}
+            "#
+        }
```

Comment:
I recommend make separate `tailwind-theme.css` and import it by `include_str!("./tailwind-theme.css")`